packages/utils/test/serialize.test.js

```diff
@@ -1,4 +1,9 @@
-import { serializeFunction, normalizeFunctions } from '../src/serialize'
+const { serializeFunction, normalizeFunctions } = require('../src/serialize')
+
+// NOTE: This test file is on purpose using CommonJS syntax to avoid code being
```

Comment:
Shall we move implementation of function to be serialized into fixtures or a sibling file without `.test`?

Reply:
Wouldn't using a fixture would make testing significantly slower and a bit more complicated.

As for separating the test functions, I would think that babel-jest would also transform those on importing. I think, haven't tested.

Any specific concerns about the current approach?

Reply:
Actually it is fine per-se but it has a side effect that test file should be pure js and inconsistent with others (imagine if we want to move it to nuxt3 repo where tests are typescript...). By fixture i meant only a single js to import and exclude from babel transform

Reply:
I'll invesitage that.

Reply:
That has worked.

One note that `normalizeFunctions` used by `serializeFunction` mutates the passed function so I had to copy imported test objects to avoid earlier tests affecting later tests.

I wonder if that's expected.

Reply:
@rchl Seems good :)